content/glossary/onion-services/contents.lr

```diff
@@ -5,7 +5,7 @@ term: onion services
 definition:
 
 Onion services (formerly known as “[hidden services](../hidden-services)”) are services (like websites) that are only accessible through the [Tor network](../tor-tor-network-core-tor).
-Onion services offer advantages over ordinary services on the non-private web, including:
+Onion services offer advantages over ordinary services on the non-private web, including: hidden IP addresses, end-to-end encrypted traffic, and automatically generated .onion addresses etc. Read more about [onion services](https://tb-manual.torproject.org/onion-services/). 
```

Comment:
Maybe we could resume the advantages presented here, and share the link for more information: https://community.torproject.org/onion-services/overview/